R-packages/covidcast/vignettes/covidcast.Rmd

```diff
@@ -25,6 +25,17 @@ devtools::install_github("cmu-delphi/covidcast", ref = "main",
                          subdir = "R-packages/covidcast")
 ```
 
+Building vignettes takes a substantial amount of time, so they are not included
+by default. If you wish to include vignettes, use this modified command
```

Comment:
```suggestion
Building the vignettes, such as this Getting Started guide, takes a substantial amount of time. They are not included
in the package by default. If you wish to include vignettes, use this modified command:
```

with appropriate line-wrapping applied

Reply:
I tested the command locally and was able to build the vignettes. Phrasing has been updated.